GVFS/GVFS.Common/HealthCalculator/EnlistmentHealthData.cs

```diff
@@ -10,7 +10,7 @@ public class EnlistmentHealthData
             int placeholderCount,
             int modifiedPathsCount,
             decimal healthMetric,
-            List<KeyValuePair<string, int>> directoryHydrationLevels)
+            List<KeyValuePair<string, EnlistmentHealthCalculator.SubDirectoryInfo>> directoryHydrationLevels)

```

Comment:
Can this just be a list of `SubDirectoryInfo`, now that `SubDirectoryInfo` has a `name` property? I was looking to see if we could remove the `.Key` and `.Value` indirections required to access the list elements.

Reply:
Yes, it can be. Before, the sorting happened after this object was constructed but that behavior changed, I'll swap it over